arches/app/templates/views/components/widgets/resource-instance-select.htm

```diff
@@ -20,6 +20,14 @@
         </div>
     </div>
 </div>
+<!-- ko if: newTileStep -->
+<div style="position: absolute; background: white;" data-bind="with: newTileStep, visible: showCard">
```

Comment:
@ryan86 can you please use classes rather than inline styles?

Also, why use `showCard` here? couldn't you instead just use `newTileStep` and set it to null when the user is done adding the new resource?

Reply:
Inline styles and references to showCard have been removed.